packages/nc-gui-v2/pages/project/index/[id].vue

```diff
@@ -0,0 +1,86 @@
+<script lang="ts" setup>
+import { onMounted } from '@vue/runtime-core'
+import type { Form } from 'ant-design-vue'
+import type { ProjectType } from 'nocodb-sdk'
+import { ref } from 'vue'
+import { useToast } from 'vue-toastification'
+import { navigateTo, useNuxtApp, useRoute } from '#app'
+import { extractSdkResponseErrorMsg } from '~/utils/errorUtils'
+import { projectTitleValidator } from '~/utils/projectCreateUtils'
+import MaterialSymbolsRocketLaunchOutline from '~icons/material-symbols/rocket-launch-outline'
+
+const loading = ref(false)
+
+const { $api, $state } = useNuxtApp()
+const toast = useToast()
+const route = useRoute()
+
+const nameValidationRules = [
+  {
+    required: true,
+    message: 'Title is required',
```

Comment:
For users, would it be better to show `Project name is required` rather than `Title is required`?

![image](https://user-images.githubusercontent.com/35857179/180166764-5bca9702-8284-4617-a59d-4308509e20a8.png)